frontend/src/APIClients/BaseAPIClient.ts

```diff
@@ -128,6 +128,9 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
   }
   // eslint-disable-next-line no-console
   if (networkError) console.log("[Network error]", networkError);
+  if (networkError === "Error: Failed to refresh access token!") {
+    location.href = '/logout';
```

Comment:
Probably just need to disable this l'inter error, since we don't have a choice here.

Reply:
Yeah I updated it so I think it should be fixed. However, I have no idea how to test this except for waiting to see if the error occurs. Do you have any ideas how I can at least force this function to run, ideally with the debugger?